.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,29 +1,15 @@
 <!--
-Thanks for contributing a pull request! Please ensure you have taken a look at
-the contribution guidelines: https://github.com/pydicom/pydicom/blob/master/CONTRIBUTING.md#contributing-pull-requests
+Please prefix your PR title with [WIP] for PRs that are in progress and change
+to [MRG] when you consider them ready for final review.
 -->
-#### Reference Issue
-<!-- Example: Fixes #1234 -->
 
+#### Describe the changes
+The related issue or a description of the bug or feature that this PR addresses.
 
-#### What does this implement/fix? Explain your changes.
-<!--
-Please summarize the key points of the reference issue, problem or contribution
-to facilitate reviewing task. Of course reviewers can always refer to the
-original issue but facilitating the reviewing process is much appreciated.
--->
-
-#### Any other comments?
-<!--
--->
-
-<!--
-Please be aware that we are a loose team of volunteers so patience is
-necessary; assistance handling other issues is very welcome. We value
-all user contributions, no matter how minor they are. If we are slow to
-review, either the pull request needs some benchmarking, tinkering,
-convincing, etc. or more likely the reviewers are simply busy. In either
-case, we ask for your understanding during the review process.
-
-Thanks for contributing!
--->
+#### Tasks
+- [ ] Unit tests added that reproduce the issue or prove feature is working
+- [ ] Fix or feature added
+- [ ] Documentation updated (if relevant)
+  - [ ] No warnings during build
+  - [ ] Preview link (CircleCI -> Artifacts -> `[...]/_build/html/index.html`)
```

Comment:
Is the preview link instruction clear enough?

Reply:
Yes, I think so. I actually used to manually add `#artifacts` to the URL until now...